guides/common/modules/snip_note-custom-templates-unsupported.adoc

```diff
@@ -0,0 +1,5 @@
+[NOTE]
+====
+{Team} can only provide support for the original unedited templates.
```

Comment:
```suggestion
{Team} only provides support for the original unedited templates.
```

Reply:
See also https://docs.orcharhino.com/or/docs/sources/guides/deploying_an_internal_application.html#Using_Custom_Templates for inspiration.

Reply:
Okay, but I'd prefer to keep it simple for this context. I think it's sufficient.